get_http_test.go

```diff
@@ -159,6 +163,38 @@ func TestHttpGetter_file(t *testing.T) {
 	assertContents(t, dst, "Hello\n")
 }
 
+var rangeReqs = []struct {
```

Comment:
I would put this directly in your test case to keep it scoped to just that test.

Reply:
Can do. Makes sense.